patches/expo-image+2.4.1.patch

```diff
@@ -58,10 +47,10 @@ index 4bab386..a41d924 100644
  import SDWebImageSVGCoder
  
  public final class ImageModule: Module {
-@@ -211,7 +210,6 @@ public final class ImageModule: Module {
+@@ -230,7 +229,6 @@ public final class ImageModule: Module {
+ 
    static func registerCoders() {
-     // By default Animated WebP is not supported
-     SDImageCodersManager.shared.addCoder(SDImageAWebPCoder.shared)
+     SDImageCodersManager.shared.addCoder(WebPCoder.shared)
```

Comment:
@enahum i don't think i should be adding this.  But this is WebPCoder.shared instead of SDImageAWebPCoder.shared.

Reply:
Nevermind.  I got confused with the `-` of the actual patch and also github `-`.

Apple added native WebP support in iOS 14, so SDImageAWebPCoder uses the system decoder. However, for some animated WebP images, the libwebp decoder (SDImageWebPCoder) performs better. So Expo created this wrapper to:

1. Default to Apple's native decoder (useAppleWebpCodec = true)
2. Allow developers to opt-out per-image with useAppleWebpCodec={false} when they encounter performance issues with animated WebPs